OpenChange/RTFHandler.m

```diff
@@ -700,9 +965,238 @@ - (void) parsePicture
   [self parseIgnoringEverything];
 }
 
-//
-//
-//
+
+// todo:  This keyword is only valid in the RTF header section right after the \ansi, \mac, \pc or \pca keyword.
+inline static void parseAnsicpg (BOOL hasArg, int arg, const unsigned short **out_default_char)
+{
+  NSString *key;
+  const unsigned short *res;
+
+  if (!hasArg)
+    return;
+  key = [NSString stringWithFormat: @"anscicpg%i", arg];
+  res =  NSMapGet(_charsets, key);
+  if (res)
+    *out_default_char = res;
+}
```

Comment:
Just guessing... Why don't you return here the `const unsigned short` having 0 as on error value?